billy-andorra/src/test/java/com/premiumminds/billy/andorra/test/services/export/TestADCreditNotePDFTransformer.java

```diff
@@ -169,11 +169,10 @@ private ADCreditNoteEntity generateESCreditNote(PaymentMechanism paymentMechanis
 
         this.createSeries(reference, "AC");
 
-        ADCreditNoteEntity creditNote = null;
-        creditNote = (ADCreditNoteEntity) services.issueDocument(
+        ADCreditNoteEntity creditNote = (ADCreditNoteEntity) services.issueDocument(
             new ADCreditNoteTestUtil(ADAbstractTest.injector).getCreditNoteBuilder(reference), params);
```

Comment:
In order to make the code more readable, i'd suggest seperating each issueDocument parameter per line